packages/graphql/src/translate/translate-read.ts

```diff
@@ -136,6 +136,26 @@ export function translateRead(
     let connectionPreClauses: Cypher.Clause | undefined;
 
     if (isRootConnectionField) {
+        const hasConnectionOrdering = resolveTree.args.first || resolveTree.args.after || resolveTree.args.sort;
+        if (hasConnectionOrdering) {
+            const afterInput = resolveTree.args.after as string | undefined;
```

Comment:
can `resolveTree.args.after` be sth other than string if defined? I see the old code is explicitly checking its type before passing it to `cursorToOffset`

Reply:
Not as far as I'm aware - it is a type `String` in the schema, so that should give it enough type safety.